packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -135,7 +140,8 @@ export const buildModules = async (
           },
         }),
       ],
-    });
+    };
+    await mergeConfig(viteConfig, override);
```

Comment:
You're not executing build anymore. You're only executing the merging of the configs. You need to merge them and then pass that config to build.

Reply:
Yes ur right. Didn't notice since npm run build passed (but the serverless functions weren't built). fixed!